app/forms/EndorsementPlaceFormProvider.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.mappings.Mappings
+import javax.inject.Inject
+import play.api.data.Form
+
+class EndorsementPlaceFormProvider @Inject() extends Mappings {
+
+  def apply(prefix: String): Form[String] =
```

Comment:
With this being a text entry field similar to contact person name page we cannot allow certain chars such as '<'or '<'  would break the code when sending xml to the BE. Should we not have the same validation in as contact person name for chars where we only allow certain chars as used the name form provider?
            regexp(stringFieldRegex, s"$prefix.error.invalid")

Reply:
I was thinking I wouldn't be able to do this until we get the content through but if you think it's likely to be the same as that one, happy to add it in and we can always change it when the content comes through